content/docs/user-guide/scripting/script-canvas/programmer-guide/behavior-context.md

```diff
@@ -97,105 +247,112 @@ public:
+}
+```
 
-{{< todo issue="https://github.com/o3de/o3de.org/issues/992" >}}
-This example is outdated. It refers to a legacy component that no longer exists.
-{{< /todo >}}
+Once compiled, these event handlers are available to Script Canvas from the `LightNotification` node.
 
-Objects that are reflected to the behavior context are also available in Script Canvas. Objects become available in the form of variables.
+![Light notification node](/images/user-guide/scripting/script-canvas/behavior-context-ebus-light-notification-node.png)
 
-The Light component does not provide any objects, but the `PhysicsComponent` provides an example of an object reflected to the behavior context. The `Collision` class is reflected as a behavior context object that you can access through the use of variables in Script Canvas. You can find the following source code in the file `dev\Gems\LmbrCentral\Code\Source\Physics\PhysicsComponent.cpp`.
+## Example: Data types
 
-```cpp
-using Collision = PhysicsComponentNotifications::Collision;
-// Information about a collision event
-behaviorContext->Class<Collision>()
-    ->Attribute(AZ::Script::Attributes::Storage, AZ::Script::Attributes::StorageType::Value)
-    ->Property("entity", BehaviorValueProperty(&Collision::m_entity))
-    ->Property("position", BehaviorValueProperty(&Collision::m_position))
-    ->Property("normal", BehaviorValueProperty(&Collision::m_normal))
-    ->Property("impulse", BehaviorValueProperty(&Collision::m_impulse))
-    ->Property("velocities", BehaviorValueGetter(&Collision::m_velocities), nullptr)
-    ->Property("masses", BehaviorValueGetter(&Collision::m_masses), nullptr)
-    ->Property("surfaces", BehaviorValueGetter(&Collision::m_surfaces), nullptr)
-     ;
-```
+To make a custom data type available for use as a variable in Script Canvas, you can reflect it using the behavior context's Class builder. The new type is also available to pass as a parameter to functions and events.
 
-{{< note >}}
-<!-- During the preview release of Script Canvas, -->Properties must provide both getters and setters to be accessible on a Script Canvas node. Containers such as vectors are currently not supported. For this reason, velocities, masses, and surfaces do not provide a setter.
-{{< /note >}}
+This example uses the `BoxShapeConfig` class as an example. This class is defined in the file `Gems\LmbrCentral\Code\include\LmbrCentral\Shape\BoxShapeComponentBus.h` and reflected in `Gems\LmbrCentral\Code\Source\Shape\BoxShapeComponent.cpp`.
 
-Most object variables are set as a result of an event. In the case of the preceding `Collision` example, the `Collision` variable is returned by the `OnCollision` event.
+A data type must be reflected to both the `SerializeContext` and the `BehaviorContext`. The serialization context enables the data type to be stored and read from a file, and the behavior context allows it to be bound to the scripting system.
```

Comment:
To match the style on this page I might suggest changing this to "...reflected to both the serialization context and the behavior context.

Alternatively, this could refer to to the codeblock below and read "...reflected in both `serializeContext` and `behaviorContext`".